### ISSUE TITLE
feature request: change theme for new window

### ISSUE THREAD

**Issue by dchandran** [EliverLara/terminator-themes#23]
I use terminator extensively. Thank you for these themes. However, something that makes me more productive is to use a different theme so that I can quickly identify different windows. It would be nice if the theme changes automatically when I split left/down.

**Comment by efka-9** [comment]
I agree. It would be great if theme could change on split screen action.
